docs/userguide/dev/ci.md

```diff
@@ -0,0 +1,170 @@
+
+## Azure Pipelines Configuration
+
+ICU also uses [Azure Pipelines](https://azure.microsoft.com/en-us/products/devops/pipelines) to run CI jobs.
+
+### Workflow files
+
+Workflow files are in YAML form stored at `.ci-builds/`.
+
+### UI Dashboard
+
+All Azure piplines show up in the 
+[Azure Piplines ICU project dashboard](https://dev.azure.com/ms/icu/),
+specifically in the 
+[Pipelines page](https://dev.azure.com/ms/icu/_build).
+
+After clicking on a specific pipeline,
+all of the instances/runs for that pipeline appear.
+All of the runs can be filtered in the Branch filter by the destination branch name (ex: `main`) or the PR number (ex: `2874`).
+
+### Configuration
+
+#### Initial Setup with Github.
+
+The upstream Github repo needs to be connected to Azure Pipelines in order for Azure Pipelines to listen for and trigger new pipelines based on events occuring in Github,
+and to return the status back to Github.
+This configuration is started by ensuring that the upstream ICU repo `unicode-org/icu` has the Azure Pipelines 3rd party app installed from the Github Marketplace.
+
+Once installed, the app will appear in the repo's Github "Settings" page under "Integrations" > "Github Apps".
+
+In order to configure a newly pipeline in Azure using the Github app for Azure Pipelines, you must do:
+
+1. Create and check-in a new YAML file in the icu repo at `.ci-builds/`
+2. In the repo settings, go to "Github Apps"
+3. In the Installed GitHub Apps section, click option to configure "Azure Pipelines".
+4. A page to update Azure Pipeline's repository access apears next. Click on "Update access".
+5. Next a page to authenticate with your Microsoft credentials appear. Sign in with Microsoft credentials.
+6. Select ADO org as `ms` and project as `icu`, click on continue.
+7. After authentication from Azure and Github, you come to the new pipeline wizard.
+8. Select repo as `unicode-org/icu` and select "Existing Azure Pipelines YAML file" and choose the yaml file created in step #1
+9. Review YAML file and click save. You will find a new pipeline created with name `unicode-org.icu`. Rename it to a more appropriate name
+
+***The pipline should now run as per the YAML rules and would be visible from Github settings for branch protection.***
+
+
+#### Jobs
+
+Jobs can be added, removed, and edited merely by editing the YAML workflow files.
+
+The syntax for Azure Pipelines workflows is very similar to Github Actions, including the YAML format.
+However, there are noteworthy differences in functionality and the expression of equivalent configurations aross the systems and syntaxes.
+
+#### Conditional Triggers
+
+Conditional triggers can be configured for Azure Pipelines similarly to Github Actions.
+
+Note: the triggers for merges to a branch (ex: `main`) may need to be duplicated into a separate trigger section for Pull Requests because they seem to be handled differently.
```

Comment:
```suggestion
Note: The triggers for merges to a branch (ex: `main`) may need to be duplicated into a separate trigger section for Pull Requests because they seem to be handled differently.
```